src/utilities/index.js

```diff
@@ -60,3 +60,13 @@ export {
 
 // Asserts that a string is a valid GraphQL name
 export { assertValidName } from './assertValidName';
+
+// Helpful functions to compare two GraphQLSchemas and detect breaking changes.
+export {
+  findBreakingChanges,
```

Comment:
> Unit tests should be importing from that file directly - not from this package module.

Turns out the test was already directly importing from the file.

> If there real need in the future then we could export these then - otherwise I'd prefer to keep the surface area of the exported API smaller and preserve the option value to change the implementation of findRemovedTypes in the future.

That's a very good point. Updated so that only `findBreakingChanges` is exported.